README.md

```diff
@@ -1,165 +1,57 @@
+<!-- prettier-ignore -->
+
 [![Join the chat at https://gitter.im/scalecube-js/Lobby](https://badges.gitter.im/scalecube-js/Lobby.svg)](https://gitter.im/scalecube-js/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-> **NOTICE** versions 0.0.x are experimental without LTS or the API and behavior might change from patch to patch 
+> **NOTICE** versions 0.0.x are experimental without LTS or the API and behavior might change from patch to patch
 
 # Scalecube-js
-Microservices library inspired by the java scalecube-services. scalecube-js provides an abstraction toolkit to provision and consume microservices as such the interaction model is by a service api avoiding tight-coupeling between service implemntation and technology.
 
-With scalecube-js decouple service by interface and agnostic to the location of service implementaion, location, and technology.
-in the roadmap scalecube-js will support client side and server side. as such services may be:
-* Deployable unit of same application and technology
-* Deployable unit of diffrent application and diffrent technology
-* Diffrent service workers in a browser
-* Node-js services on same host
-* Node-js services on diffrent host communicating via api.
+Microservices library inspired by the java scalecube-services. scalecube-js provides an abstraction toolkit to provision and consume microservices as such the interaction model is by a service api avoiding tight-coupling between service implementation and technology.
```

Comment:
Microservices library inspired by the java scalecube-services
@scalecube/microservices is the equivalent to (scalecube-services)[url] (Java implementation)